README.md

```diff
@@ -29,10 +29,11 @@ Templates:
 * [Decision record template by Michael Nygard](templates/decision-record-template-by-michael-nygard/index.md)
 * [Decision record template for Alexandrian pattern](templates/decision-record-template-for-alexandrian-pattern/index.md)
 * [Decision record template for business case](templates/decision-record-template-for-business-case/index.md)
-* [Decision record template for MADR](templates/decision-record-template-madr/index.md)
+* [Decision record template in Markdown](templates/decision-record-template-madr/index.md)
```

Comment:
Had a discussion with @socadk:

How about "Decision record template of the MADR project"?